plexpy/notifiers.py

```diff
@@ -2754,32 +2724,82 @@ class HIPCHAT(object):
     def __init__(self):
         self.apiurl = plexpy.CONFIG.HIPCHAT_URL
         self.color = plexpy.CONFIG.HIPCHAT_COLOR
-        self.incl_subject = plexpy.CONFIG.HIPCHAT_INCL_SUBJECT
         self.emoticon = plexpy.CONFIG.HIPCHAT_EMOTICON
+        self.from_subject = plexpy.CONFIG.HIPCHAT_FROM_SUBJECT
+        self.incl_pmslink = plexpy.CONFIG.FACEBOOK_INCL_PMSLINK
+        self.incl_poster = plexpy.CONFIG.FACEBOOK_INCL_POSTER
+        self.incl_subject = plexpy.CONFIG.HIPCHAT_INCL_SUBJECT
 
-    def notify(self, message, subject):
+    def notify(self, message, subject, **kwargs):
         if not message or not subject:
             return
-        
+
+        data = {'notify': 'false'}
+
+        text = message.encode('utf-8')
+
         if self.incl_subject:
-            text = subject.encode('utf-8') + ': ' + message.encode('utf-8')
-        else:
-            text = message.encode('utf-8')
+            text = subject.encode('utf-8') + ': ' + text
 
         if self.emoticon:
             text = self.emoticon + ' ' + text
 
-        data = {'message': text,
-                'notify': 'false',
-                'message_format': 'text'}
-
         if self.color:
             data['color'] = self.color
 
+        if self.from_subject:
```

Comment:
If `HIPCHAT_INCL_SUBJECT` and `HIPCHAT_FROM_SUBJECT` are both checked, the subject line shows up twice. I might be better to only have one setting.

If `HIPCHAT_INCL_SUBJECT` is enabled then have the subject as the "From" line?